python/tests/test_erst.py

```diff
@@ -0,0 +1,141 @@
+import unittest
+import sys
+import numpy as np
+import io
+
+from opm.io.ecl import ERst, eclArrType
+try:
+    from tests.utils import test_path
+except ImportError:
+    from utils import test_path
+
+
+class TestERst(unittest.TestCase):
+    
+    def test_reportSteps(self):
+
+        info_str = "List of report steps\n\nseqnum =   37 Date: 06-Jan-2016\nseqnum =   74 Date: 10-Jan-2017\n"
+
+        rst1 = ERst(test_path("data/SPE9.UNRST"))
+
+        self.assertTrue(repr(rst1) == info_str)
+
+        self.assertTrue(rst1.has_report_step_number(37))
+        self.assertTrue(rst1.has_report_step_number(74))
+
+        self.assertRaises(ValueError, rst1.load_report_step, 137 );
```

Comment:
The most common way to express these tests is:
```python
with self.assertRaises(ValueError):
      rst1.load_report_step( 137 )
```

Reply:
@joakim-hove can you please let me know if this will be merged?  When do you expect to finish reviewing this PR ?